src/thor/matrix_action.cc

```diff
@@ -36,7 +36,8 @@ std::string thor_worker_t::matrix(Api& request) {
   auto costmatrix = [&](const bool has_time) {
     return costmatrix_.SourceToTarget(request, *reader, mode_costing, mode,
                                       max_matrix_distance.find(costing)->second, has_time,
-                                      options.date_time_type() == Options::invariant);
+                                      options.date_time_type() == Options::invariant,
+                                      options.shape_format());
```

Comment:
actually why do we even pass these things here.. why also have the full `request` as argument, it's kinda unnecessary

Reply:
im sure its a result of things slowy changing over time. i appreciate resisting the urge to refactor it as part of this though i agree its stupid :smile:

Reply:
this is my fail anyways:)